src/tools/driver/help_text.dart

```diff
@@ -33,6 +35,16 @@ Supported things:
   poi
```

Comment:
Add logcat to this help text too? Is logcat a good and user-friendly name for this?